flask_ipblock/__init__.py

```diff
@@ -3,15 +3,28 @@
 
 
 class IPBlock(object):
-    def __init__(self, app, read_preference=None):
+
+    def __init__(self, app, read_preference=None, cache_size=None,
+                 cache_ttl=None):
         """
         Initialize IPBlock and set up a before_request handler in the
         app.
 
         You can override the default MongoDB read preference via the
         optional read_preference kwarg.
+
+        You can limit the impact of the IP checks on your MongoDB by
+        maintaining a local in-memory LRU cache. To do so, specify its
+        size via cache_size and the TTL (in seconds) via cache_ttl.
         """
         self.read_preference = read_preference
+
+        self.cache = None
```

Comment:
Can just put that in an `else` clause below (no need to double assign to `self.cache` if there's a cache)